wsrichmemo.pas

```diff
@@ -153,6 +210,48 @@ class function TWSCustomRichMemo.CanPasteFromClipboard(
   Result := true;
 end;
 
+class function TWSCustomRichMemo.HandleKeyDown(const AWinControl: TWinControl;
+  key: word): boolean;
```

Comment:
Is this TWSCustomRichMemo.HandleKeyDown() actually Gtk2 handle key?
I can see it's has a different implementation for Qt5. 

Should the default implementation return false (just like Win version does) and this implementation move to Gtk2?

Reply:
It is, I first implemented the gtk part without knowing that qt already do some things (and I have not touched the cocoa part yet), the intention was to handle bullet and numbered lists managing in a widgetset independent way, but it turned out to be not necessary, I hope cocoa also will not need it and then it will be moved to the gtk side.

Reply:
how about making a qt5 implentation first?

cocoa does do everything by itself, just like Windows and Qt.

Reply:
nice!, then that solves it. I will move everything to gtk